plugins.json

```diff
@@ -2499,6 +2499,62 @@
     },
     "updated_at": "2019-06-12T19:04:44Z"
   },
+  {
+    "homepage_url": "",
+    "icon_data": "",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-channel-export-v0.2.2.tar.gz",
+    "release_notes_url": "",
+    "labels": [
+      {
+        "name": "Enterprise (E20)",
+        "description": "This plugin only works on servers with an E20 license.",
+        "url": "https://mattermost.com/pricing/",
+        "color": ""
+      }
+    ],
```

Comment:
@itao @aaronrothschild Should this plugin be marked as `Beta` given that it's "only" `v0.2.2`?

Reply:
We could cut a `v1.0.0` if needed, but it would just be for optics -- the feature set stabilized months ago for this plugin.

Reply:
@lieut-data Let's cut v1.0, the existing features are stable and wholly solves a customer need.